composer.json

```diff
@@ -85,7 +85,7 @@
         "symfony/browser-kit": "^4.4 || ^5.1",
         "symfony/css-selector": "^4.4 || ^5.1",
         "symfony/filesystem": "^4.4 || ^5.1",
-        "symfony/maker-bundle": "^1.25",
+        "symfony/maker-bundle": "^1.29",
```

Comment:
What does this have to do with Psalm or deprecations?

Reply:
Removed